src/components/tables/RowActions/RowSelector.tsx

```diff
@@ -0,0 +1,117 @@
+import ArrowDropDownIcon from '@mui/icons-material/ArrowDropDown';
+import CheckBoxOutlineBlankIcon from '@mui/icons-material/CheckBoxOutlineBlank';
+import IndeterminateCheckBoxIcon from '@mui/icons-material/IndeterminateCheckBox';
+import { Button, ButtonGroup, Menu, MenuItem, Stack } from '@mui/material';
+import Delete from 'components/tables/RowActions/Delete';
+import Materialize from 'components/tables/RowActions/Materialize';
+import {
+    SelectableTableStore,
+    selectableTableStoreSelectors,
+} from 'components/tables/Store';
+import { useZustandStore } from 'hooks/useZustand';
+import { useState } from 'react';
+import { FormattedMessage, useIntl } from 'react-intl';
+
+export interface RowSelectorProps {
+    showMaterialize?: boolean;
+}
+
+function RowSelector({ showMaterialize }: RowSelectorProps) {
```

Comment:
Used to select all/none rows. Also houses the button group for taking actions of the selected rows.